.env.example

```
@@ -0,0 +1,6 @@
+API_BASE_URL=http://localhost:5010/graphql
+
+# Project R Newsletter format ID
+NEWSLETTER_ID=12345
+
+REACT_APP_SG_FONT_STYLES={"serifTitle":{"fontFamily":"serif","fontWeight":900},"serifRegular":{"fontFamily":"serif"},"serifItalic":{"fontFamily":"serif","fontStyle":"italic"},"serifBold":{"fontFamily":"serif","fontWeight":700},"serifBoldItalic":{"fontFamily":"serif","fontWeight":700,"fontStyle":"italic"},"sansSerifRegular":{"fontFamily":"sans-serif"},"sansSerifItalic":{"fontFamily":"sans-serif","fontStyle":"italic"},"sansSerifMedium":{"fontFamily":"sans-serif","fontWeight":500},"monospaceRegular":{"fontFamily":"monospace"},"cursiveTitle":{"fontFamily":"sans-serif","fontWeight":500,"fontStyle":"italic"}}
```

```suggestion
SG_FONT_STYLES={"serifTitle":{"fontFamily":"serif","fontWeight":900},"serifRegular":{"fontFamily":"serif"},"serifItalic":{"fontFamily":"serif","fontStyle":"italic"},"serifBold":{"fontFamily":"serif","fontWeight":700},"serifBoldItalic":{"fontFamily":"serif","fontWeight":700,"fontStyle":"italic"},"sansSerifRegular":{"fontFamily":"sans-serif"},"sansSerifItalic":{"fontFamily":"sans-serif","fontStyle":"italic"},"sansSerifMedium":{"fontFamily":"sans-serif","fontWeight":500},"monospaceRegular":{"fontFamily":"monospace"},"cursiveTitle":{"fontFamily":"sans-serif","fontWeight":500,"fontStyle":"italic"}}
```

The `REACT_APP_` is only necessary in the styleguide docs because it's powered by create react app.